src/app/js/admin/upload/index.js

```diff
@@ -0,0 +1,33 @@
+import { createAction, handleActions } from 'redux-actions';
+
+export const UPLOAD_FILE = 'UPLOAD_FILE';
+export const UPLOAD_FILE_ERROR = 'UPLOAD_FILE_ERROR';
+export const UPLOAD_FILE_PENDING = 'UPLOAD_FILE_PENDING';
```

Comment:
You don't need pending, set the status to 'pending' when `UPLOAD_FILE` is triggered.